recipes/netgen/python.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
This patch isn't used in the recipe?

Reply:
yes I replaced it with python1.patch. I wasn't sure if it works with the updated patch so the old patch is still there. I am going to delete this patch.

Reply:
Note: since been changed back to use this patch.